operator/values.yaml

```diff
@@ -109,3 +109,9 @@ resources:
   requests:
     cpu: 100m
     memory: 20Mi
+
+# presignedUrlExpirationTimes -- Duration how long presigned urls are valid
+presignedUrlExpirationTimes:
+    scans: "24h"
```

Comment:
i think it would be better to call it `scanners` here, `scans` usually refer to the Scan CRD